domains/staging.json

```diff
@@ -0,0 +1,8 @@
+{
+    "owner": {
+        "username": "pmNiko"
+    },
+    "record": {
+        "CNAME": "https://pmniko.github.io/"
```

Comment:
Little content on site.

CNAME records must be hostnames without any protocols or file paths.